docs/content/en/docs/references/cli/_index.md

```diff
@@ -104,6 +104,7 @@ Options:
       --cache-file='': Specify the location of the cache file (default $HOME/.skaffold/cache)
   -d, --default-repo='': Default repository value (overrides global config)
       --enable-rpc=false: Enable gRPC for exposing Skaffold events (true by default for `skaffold dev`)
+      --file-output='': Filename to write build images to
```

Comment:
Do we still need "Used in conjunction with --quiet flag" for `-o, --output`? I guess from now on that flag even applies without quiet mode?!

Reply:
I think that should still be there, but be updated to say "Used in conjunction with --quiet or --file-output", since those are the only two flags whose output it effects.